core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -90,6 +92,54 @@ package object config {
     .toSequence
     .createWithDefault(Nil)
 
+  // Blacklist confs
+  private[spark] val BLACKLIST_ENABLED =
+    ConfigBuilder("spark.blacklist.enabled")
+      .booleanConf
+      .createOptional
+
+  private[spark] val MAX_TASK_ATTEMPTS_PER_EXECUTOR =
+    ConfigBuilder("spark.blacklist.task.maxTaskAttemptsPerExecutor")
+      .intConf
+      .createWithDefault(1)
```

Comment:
I'll file a separate JIRA for this once I think it through a little more. My preference is to see error strings and default values in config files instead of buried in the code. It can be better for readability, and makes it easier to support internationalization. So, for example, instead of createWithDefault(1), do something like createWithDefault(DEFAULT_TASK_ATTEMPTS_PER_EXECUTOR), with some other config file containing a line DEFAULT_TASK_ATTEMPTS_PER_EXECUTOR = 1. But there are bigger fish to fry and I see no reason at all to hold up this change for issues like this. I hope this is more clear than my terse explanation above.